voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -117,169 +123,184 @@ public void awaitRegistrationWithAll(long timeout, TimeUnit unit) throws Interru
               }
```

Comment:
@myronkscott I couldn't find any usage of `awaitRegistrationWithAll` API, any idea where this is used and what should be the behavior when topology changes?

Reply:
I think it is important to register with all before voting with the active.